android/src/main/kotlin/clean/news/ui/item/detail/ItemDetailView.kt

```diff
@@ -46,17 +47,19 @@ class ItemDetailView : RelativeLayout {
 		super.onFinishInflate()
 
 		toolbar.setNavigationIcon(R.drawable.abc_ic_ab_back_mtrl_am_alpha)
-
+		toolbar.inflateMenu(R.menu.item_detail_view)
 		commentRecyclerView.layoutManager = LinearLayoutManager(context)
 		commentRecyclerView.adapter = adapter
 	}
 
 	override fun onAttachedToWindow() {
 		super.onAttachedToWindow()
-
 		val sinks = model.setUp(Sources(
 				toolbar.navigationClicks(),
-				Observable.empty()
+				Observable.empty(),
+				toolbar.itemClicks()
+						.filter {it.itemId == R.id.item_share }
+						.map { Unit }
```

Comment:
I think if we change the source observable to `Observable<Any>`, we can avoid the mapping. Sorry,  I didn't catch this last round of comments.